src/hotspot/share/runtime/thread.cpp

```diff
@@ -3148,8 +2947,19 @@ void JavaThread::print_on_error(outputStream* st, char *buf, int buflen) const {
   return;
 }
 
+
 // Verification
 
+void JavaThread::frames_do(void f(frame*, const RegisterMap* map)) {
+  // ignore is there is no stack
```

Comment:
typo - s/is there/if there/

Reply:
Fixed.  That code block showed up in the diff since I moved the function just above the one that called it.